pkg/minikube/cluster/cluster_test.go

```diff
@@ -50,3 +53,52 @@ func TestLocalBoot2DockerURL(t *testing.T) {
 		t.Fatalf("Expected URL : %s", localISOUrl)
 	}
 }
+
+func TestHostGetLogs(t *testing.T) {
+	api := tests.NewMockAPI()
```

Comment:
Yes @hferentschik , was only trying to verify the log commands to be received by ssh server. 

To really test logs are working, we need to put test in integration test.